### DISCUSSION TITLE
Noob Q: How to I upgrade from 1.1.4 to 1.1.5

### DISCUSSION THREAD

**Discussion by rcamarda390** [Chia-Network/chia-blockchain#4420] [Support]
I have a raspberry Pi 4 running ubuntu and chia installed. 
I see that there is new chia out, and I am having trouble getting it upgraded.
I tried a lot of commands between git and apt-install (sorry I did not document), and I thihk it was a git command that actually did something.
However, when I tried 'chia version' it is still on shows 1.1.4.
Can i get help on what is the way to get new version (branch?) from github so I can get on current version.
Lemme know if there are logs that would be helpful

**Comment by ilyakooo0** [comment]
You didn't mention: did you actually look at the updating documentation?

Search for "To Update/Upgrade from previous version" in the [installation wiki](https://github.com/Chia-Network/chia-blockchain/wiki/INSTALL#ubuntudebian).

**Comment by rcamarda390** [comment]
nope, cuz I didnt find it while googling. THANKS!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
